docs/fields.md

```diff
@@ -401,7 +401,7 @@ field.
 Directory specification can be relative, absolute,
 or part of a URL.  URL specifications should
 follow the [hashicorp URL] format.  The directory
-must contain a `kustomization.yaml` file.
+must contain a `kustomization.yaml` / `kustomization.json` file.
```

Comment:
Let's not add `kustomization.json` here since we didn't do that for other accepted files like kustomization.yml, kustomization.